integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java

```diff
@@ -176,31 +177,49 @@ public void unpinFileTest() throws Exception {
-   * WriteType.TRY_CACHE, MEM_CAPACITY_BYTES / 2);
-   *
-   * CommonUtils.sleepMs(null, CommonUtils.getToMasterHeartBeatIntervalMs(mWorkerConf) * 2 + 10);
-   *
-   * TachyonFile file1 = mTFS.getFile(fileId1); TachyonFile file2 = mTFS.getFile(fileId2);
-   * TachyonFile file3 = mTFS.getFile(fileId3);
-   *
-   * Assert.assertEquals(false, file1.isInMemory()); Assert.assertEquals(true, file2.isInMemory());
-   * Assert.assertEquals(true, file3.isInMemory()); Assert.assertEquals(MEM_CAPACITY_BYTES / 6 +
-   * MEM_CAPACITY_BYTES, mLocalTachyonCluster.getMasterInfo().getUsedBytes());
-   *
-   * InStream is = file1.getInStream(ReadType.CACHE_PROMOTE); byte[] buf = new
-   * byte[MEM_CAPACITY_BYTES / 6]; int len = is.read(buf); is.close();
-   *
-   * CommonUtils.sleepMs(null, CommonUtils.getToMasterHeartBeatIntervalMs(mWorkerConf) * 2 + 10);
-   *
-   * Assert.assertEquals(MEM_CAPACITY_BYTES / 6, len); Assert.assertEquals(true,
-   * file1.isInMemory()); Assert.assertEquals(false, file2.isInMemory()); Assert.assertEquals(true,
-   * file3.isInMemory()); Assert.assertEquals(MEM_CAPACITY_BYTES / 6 + MEM_CAPACITY_BYTES,
-   * mLocalTachyonCluster.getMasterInfo().getUsedBytes()); }
-   */
+  @Test
+  public void promoteBlock() throws Exception {
+    TachyonFile file1 =
+        TachyonFSTestUtils.createByteFile(mTFS, "/root/test1", TachyonStorageType.STORE,
+            UnderStorageType.PERSIST, MEM_CAPACITY_BYTES / 6);
+    TachyonFile file2 =
+        TachyonFSTestUtils.createByteFile(mTFS, "/root/test2", TachyonStorageType.STORE,
+            UnderStorageType.PERSIST, MEM_CAPACITY_BYTES / 2);
+    TachyonFile file3 =
+        TachyonFSTestUtils.createByteFile(mTFS, "/root/test3", TachyonStorageType.STORE,
+            UnderStorageType.PERSIST, MEM_CAPACITY_BYTES / 2);
+
+    CommonUtils.sleepMs(null, mWorkerToMasterHeartbeatIntervalMs * 3);
+
+    TachyonFile toPromote = null;
+    FileInfo file1Info = mTFS.getInfo(file1);
+    FileInfo file2Info = mTFS.getInfo(file2);
+    FileInfo file3Info = mTFS.getInfo(file3);
+
+    if (file1Info.getInMemoryPercentage() < 100) {
+      toPromote = file1;
+      Assert.assertEquals(100, file2Info.getInMemoryPercentage());
+      Assert.assertEquals(100, file3Info.getInMemoryPercentage());
+    } else if (file2Info.getInMemoryPercentage() < 100) {
+      toPromote = file2;
+      Assert.assertEquals(100, file1Info.getInMemoryPercentage());
+      Assert.assertEquals(100, file3Info.getInMemoryPercentage());
+    } else {
+      toPromote = file3;
+      Assert.assertEquals(100, file1Info.getInMemoryPercentage());
+      Assert.assertEquals(100, file2Info.getInMemoryPercentage());
+    }
+
+    FileInStream is = mTFS.getInStream(toPromote, new ClientOptions.Builder(mWorkerConf)
+        .setTachyonStoreType(TachyonStorageType.PROMOTE).build());
+    byte[] buf = new byte[MEM_CAPACITY_BYTES / 6];
+    int len = is.read(buf);
+    is.close();
+
+    CommonUtils.sleepMs(LOG, mWorkerToMasterHeartbeatIntervalMs * 3);
```

Comment:
why sleep with LOG here whye sleep without LOG above? Why we treat them differently?